src/components/ContentNode.vue

```diff
@@ -185,19 +187,20 @@ function renderNode(createElement, references) {
 
   const renderFigure = ({
     metadata: {
-      abstract,
+      abstract = [],
       anchor,
       title,
     },
     ...node
-  }) => createElement(Figure, { props: { anchor } }, [
-    ...(title && abstract && abstract.length ? [
-      createElement(FigureCaption, { props: { title } }, (
-        renderChildren(abstract)
-      )),
-    ] : []),
-    renderChildren([node]),
-  ]);
+  }) => {
```

Comment:
These changes are from another PR, right? I think in the future it would be easier to open these PRs with a dependency after the dependent PR has been merged, so that they're easier to review in isolation.